sentry/src/main/java/io/sentry/IScopes.java

```diff
@@ -312,12 +312,19 @@ default void addBreadcrumb(@NotNull String message, @NotNull String category) {
   @NotNull
   ISentryLifecycleToken pushIsolationScope();
 
-  /** Removes the first scope */
+  /**
+   * Removes the first scope and restores its parent.
+   *
+   * @deprecated please call {@link ISentryLifecycleToken#close()} on the token returned by {@link
+   *     IScopes#pushScope()} or {@link IScopes#pushIsolationScope()} instead.
+   */
+  @Deprecated
```

Comment:
Deprecating this as `close` on the lifecycle token should give better results and is also nicer to use when combining with `try(...)`.